### ISSUE TITLE
Memory numbers don't make sense?

### ISSUE THREAD

**Issue by ColtonIdle** [iglance/iGlance#174]
I'm attaching 3 images.

The first shows the iglance dropdown and the "error". These numbers don't make sense to me?

The second is activity monitor and the third is iglance again but from the iglance window.

<img width="232" alt="Screen Shot 2020-04-29 at 4 56 55 AM" src="https://user-images.githubusercontent.com/31751141/80578148-106ff200-89d6-11ea-89ad-f1c2f756d4bb.png">

<img width="375" alt="Screen Shot 2020-04-29 at 4 57 01 AM" src="https://user-images.githubusercontent.com/31751141/80578172-149c0f80-89d6-11ea-8e57-0557f1afa84c.png">

<img width="343" alt="Screen Shot 2020-04-29 at 4 57 10 AM" src="https://user-images.githubusercontent.com/31751141/80578189-17970000-89d6-11ea-9391-87dda92e5f42.png">

My machine has 64GB of ram (both activity monitor and iglance report this). iglance dropdown says I have 33GB of ram free but this is impossible if activity monitor says that I'm using 44GB?

Maybe I'm just misreading the numbers from iglance dropdown. If so, maybe there's room for improvement to align these numbers so the names are similar to what I'd find if I opened activity monitor?

**Comment by D0miH** [comment]
Hey @ColtonIdle,

ok so I investigated the issue and it seems that the "App Memory" is calculated differently than just summing up the active, compressed and wired memory. This is why iGlance is apparently not displaying numbers that are in sync with the activity monitor. 

I will include the "App Memory" in the next update. In addition to that the "free" memory is then calculated accordingly to activity monitor using:
`<total_memory> - (<app_memory> + <wired_memory> + <compressed_memory>)`

Thanks for pointing this out👍🏼

**Comment by ColtonIdle** [comment]
Yeah. My main use case of using iglance is to be able to see when I'm getting close to full memory usage. because macOS seems to start putting items into swap once you start to get full, and this is a crappy situation because once swap is used my computers slows down like crazy. Nothing will fix it until a restart.

So yeah, because able to tell "You're 85% full! macOS will start swapping in the next 5%" is the reason I use iglance.

**Comment by D0miH** [comment]
I see. With the next update the free memory should be computed correctly.

As a side note we also could implement some kind of notification if the memory exceeds a threshold which was chosen by the user.

**Comment by D0miH** [comment]
Ok this should be fixed with the next update. 

I create a new [issue](#182) for the idea with the memory notifications. But I think it will take some time until this feature will be included in the app.
